src/LanguageServer/azure-pipelines.yml

```diff
@@ -0,0 +1,53 @@
+# to create dependency on backend code build pipeline
+resources:
+  pipelines:
+  - pipeline: backendbuild
+    source: 'Backend PR Build'
+    trigger:
+      branches:
+        include: 
+        - main
+
+jobs:
+
+- job: 'BuildLanguageServer'
+  pool:
+    vmImage: 'ubuntu-latest'
+
+  steps:
+  - task: DownloadBuildArtifacts@1
+    inputs:
+      buildType: 'specific'
+      project: 'd760220e-01cd-406f-b520-62a6e0779471'
+      pipeline: '12'
+      buildVersionToDownload: 'latest'
+      downloadType: 'single'
+      artifactName: 'backend-drop1'
+      downloadPath: '$(Agent.BuildDirectory)'
+
+  - task: ExtractFiles@1
+    inputs:
+      archiveFilePatterns: '$(Agent.BuildDirectory)/backend-drop1/Diagnostics.RuntimeHost.zip'
+      destinationFolder: '$(Agent.BuildDirectory)/applensdlls'
```

Comment:
nitpik; instead of applensdlls should we rename this to go to runtimehostdlls?

Reply:
sure we can. though it's a temporary folder and isn't really used anywhere.